client/src/components/PopoverField/PopoverField.js

```diff
@@ -16,7 +16,7 @@ class PopoverField extends Component {
   /**
    * Get popup placement direction
    *
-   * @returns {String}
+   * @return {String}
```

Comment:
oops, missed this... valid JSDocs is `@returns` :P

Reply:
Oh!